neo/Network/P2P/Payloads/Block.cs

```diff
@@ -106,6 +107,15 @@ public override JObject ToJson()
             return json;
         }
 
+        public new static Block FromJson(JObject json)
+        {
+            Block block = new Block();
+            (block as BlockBase).FromJson(json);
```

Comment:
@lock9 This is a compromised way to make `FromJson` as a consistent method name for BlockBase. Because `BlockBase` is the base class, all the inherited classes (like `Block`) have to convert to `BlockBase` before they can use `FromJson` since they have the same method name.

Reply:
Maybe we can use another name like `ParseFromJson` instead of `FromJson` in BlockBase.

Reply:
Maybe if you divide it into two lines, just to make it more "clear": one for the casting and another for the "FromJson"?

Reply:
Divided😅

Reply:
Thanks. I prefer this way, but let's wait if other members agree.